hw/bsp/raspberrypi4/family.c

```diff
@@ -50,63 +58,27 @@ void USB_IRQHandler(void)
 //--------------------------------------------------------------------+
 void board_init(void)
 {
-  gpio_initOutputPinWithPullNone(18);
-  gpio_initOutputPinWithPullNone(19);
-  gpio_initOutputPinWithPullNone(20);
-  gpio_initOutputPinWithPullNone(21);
-  gpio_setPinOutputBool(18, true);
-  gpio_initOutputPinWithPullNone(42);
   setup_mmu_flat_map();
   init_caches();
 
-  // gpio_initOutputPinWithPullNone(23);
-  // gpio_initOutputPinWithPullNone(24);
-  // gpio_initOutputPinWithPullNone(25);
-  gpio_setPinOutputBool(18, false);
+  // LED
+  gpio_initOutputPinWithPullNone(LED_PIN);
+  board_led_write(true);
+
+  // Button
+  // TODO
+
+  // Uart
   uart_init();
-  gpio_setPinOutputBool(18, true);
-  gpio_setPinOutputBool(18, false);
-  for (size_t i = 0; i < 5; i++) {
-  // while (true) {
-    for (size_t j = 0; j < 1000000; j++) {
-      __asm__("nop");
-    }
-    gpio_setPinOutputBool(42, true);
-    gpio_setPinOutputBool(18, true);
-    gpio_setPinOutputBool(19, true);
-    gpio_setPinOutputBool(20, true);
-    gpio_setPinOutputBool(21, true);
-    for (size_t j = 0; j < 1000000; j++) {
-      __asm__("nop");
-    }
-    gpio_setPinOutputBool(42, false);
-    gpio_setPinOutputBool(18, false);
-    gpio_setPinOutputBool(19, false);
-    gpio_setPinOutputBool(20, false);
-    gpio_setPinOutputBool(21, false);
-  }
-  // uart_writeText("hello from io\n");
-  // gpio_setPinOutputBool(24, true);
-  // gpio_setPinOutputBool(24, false);
-  // gpio_setPinOutputBool(25, true);
-  // print();
-  // gpio_setPinOutputBool(25, false);
-  // while (true) {
-  // // for (size_t i = 0; i < 5; i++) {
-  //   for (size_t j = 0; j < 10000000000; j++) {
-  //     __asm__("nop");
-  //   }
-  //   gpio_setPinOutputBool(42, true);
-  //   for (size_t j = 0; j < 10000000000; j++) {
-  //     __asm__("nop");
-  //   }
-  //   gpio_setPinOutputBool(42, false);
-  // }
-  // while (1) uart_update();
 
   // Turn on USB peripheral.
   vcmailbox_set_power_state(VCMAILBOX_DEVICE_USB_HCD, true);
 
+  // Timer 1/1024 second tick
+  SYSTMR->CS_b.M1 = 1;
+  SYSTMR->C1 = SYSTMR->CLO + 977;
+  BP_EnableIRQ(TIMER_1_IRQn);
```

Comment:
mentioned in https://github.com/hathach/tinyusb/pull/1163#issuecomment-960473264